### PR TITLE
T20373

### PR DESCRIPTION
**Pull request by GregNing**
https://redmine.techbang.com/issues/20373

### CODE REVIEW

**Comment by GregNing** [comment]
https://redmine.techbang.com/issues/20373

File: app/controllers/groups_controller.rb [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
-    
-end
\ No newline at end of file
+  before_action :find_group, only: %i[edit show update destroy]
+
+  def index
+    @groups = Group.all
+  end
+
+  def show; end
```
Comment (myohmy10420) [inline app/controllers/groups_controller.rb]: 我們的程式碼還不用到這麼簡略
def show
end
即可

File: app/controllers/groups_controller.rb [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+  def create
+    @group = Group.new(group_params)
+    if @group.save
+      redirect_to groups_path
+    else
+      render :new
+    end
+  end
+
+  def edit; end
```
Comment (myohmy10420) [inline app/controllers/groups_controller.rb]: 我們的程式碼還不用到這麼簡略
def edit
end
即可

File: app/views/groups/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        <thead>
+          <tr>
+            <th>No.</th>
+            <th>Title</th>
+            <th>Description</th>
+            <th>Options</th>
+          </tr>
+        </thead>
+        <tbody>
+        <% @groups.each do |group| %>
```
Comment (myohmy10420) [inline app/views/groups/index.html.erb]: 這裡開始我們會再縮一排

Comment (GregNing) [reply app/views/groups/index.html.erb]: solve [e8f91e2](/GregNing/rails-101/pull/1/commits/e8f91e2881f63dd073c1dab8163aa3be2ac97e54)

File: app/controllers/groups_controller.rb (hunk 2 above)
Comment (GregNing) [reply app/controllers/groups_controller.rb]: solve [e8f91e2](/GregNing/rails-101/pull/1/commits/e8f91e2881f63dd073c1dab8163aa3be2ac97e54)

File: app/controllers/groups_controller.rb (hunk 1 above)
Comment (GregNing) [reply app/controllers/groups_controller.rb]: solve [e8f91e2](/GregNing/rails-101/pull/1/commits/e8f91e2881f63dd073c1dab8163aa3be2ac97e54)

**Pull request merged**